src/ditto/literals.py

```diff
@@ -123,3 +125,12 @@
     "FP8_PER_CHANNEL_PER_TOKEN",
     "INT8",
 ]
+
+LogLevelLiteral = Literal[
+    50,  # CRITICAL,FATAL
+    40,  # ERROR
+    30,  # WARNING
+    20,  # INFO
+    10,  # DEBUG
+    0,  # NOTSET
+]
```

Comment:
How about making this literal as
```suggestion
LogLevelLiteral = Literal[
    "CRITICAL",
    "FATAL",
    "ERROR",
    "WARNING",
    "INFO",
    "DEBUG",
    "NOTSET",
]
```
and make the `set_logger_level` context to use predefined mapping like the following?
```python
{
    "CRITICAL": logging.CRITICAL,
    "FATAL": logging.FATAL,
    "ERROR": logging.ERROR,
    "WARNING": logging.WARNING,
    "INFO": logging.INFO,
    "DEBUG": logging.DEBUG,
    "NOTSET": logging.NOTSET,
}
```
Then we won't need to import the logging module every time we call the `set_logger_level`